install.sh

```diff
@@ -69,9 +69,6 @@ if [ "${DRIVER_KIND}" == "grid" ]; then
     systemctl stop nvidia-gridd || true
```

Comment:
based on the comment above, we don't need this STOP and RESTART command for the `nvidia-gridd` service anymore, since this was simply needed to allow the `cp` to work.

Reply:
REmoved